### PR TITLE
Fat body

### LINKED ISSUE #64: embryonic fat body not an embryonic/larval fat body?
"embryonic fat body" (FBbt:0005710) is_a "fat body" (FBbt:00005066) at the same level that "embryonic/larval fat body" (FBbt:00005711) is_a "fat body". This seems wrong, as there is no connection from "embryonic/larval fat body" to "embryonic fat body". So if I search for example for "embryonic/larval fat body AND ancestors" I don't get "embryonic fat body". It seems to me that "embryonic fat body" is_a ""embryonic/larval fat body". Maybe I don't have that quite right, but there ought to be a path linking embryonic fat body and embryonic/larval fat body. (In general, I think there are still a lot of issues in places with the whole 'embryonic' vs. 'embryonic/larval' thing.

### PR DESCRIPTION
**Pull request by Clare72**
fixes #64 
Slight restructuring of embryonic/larval fat body terms and improved definitions.

### CODE REVIEW

File: fbbt/src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -53671,10 +53671,10 @@ is_a: FBbt:00007116 ! presumptive embryonic/larval system
 [Term]
 id: FBbt:00005710
 name: embryonic fat body
-def: "Fat body of the embryo." [FBC:SPR]
+def: "Fat body of the embryo from stage 16/17. This structure is a single-cell thick, lateral ribbon with dorsal fingers extending from the posterior region. Holes within the ribbon are formed by the intrusion of various organs (Hoshizaki et al., 1994)." [FBC:CP, FBC:SPR, FlyBase:FBrf0073371]
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Probably worth checking usage in curation. I suspect you'll find phenotype and expression curation with specified stages before S16.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -53671,10 +53671,10 @@ is_a: FBbt:00007116 ! presumptive embryonic/larval system
 [Term]
 id: FBbt:00005710
 name: embryonic fat body
-def: "Fat body of the embryo." [FBC:SPR]
+def: "Fat body of the embryo. This structure is a single-cell thick, lateral ribbon with dorsal fingers extending from the posterior region. Holes within the ribbon are formed by the intrusion of various organs (Hoshizaki et al., 1994)." [FBC:CP, FBC:SPR, FlyBase:FBrf0073371]
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Add stage specificity here.  Based on def: 'fat body primordium' applies before Stage 16 ("the embryonic fat body is morphologically identifiable at stage 16 (Hoshizaki et al., 1994)") => embryonic fat body applies during S16 & 17.  Suspect you'll find lots of cases in FB where 'embryonic fat body' is used for earlier stages.  Probably worth flagging to Sian.

**Comment by dosumis** [comment]
I think I accidentally closed the review.  Apologies.  Anyway - let's keep this open for further discussion.  Looks like everything else should make the release.

**Comment by Clare72** [comment]
Out of date with current fbbt- better to redo at some point rather than messy merge

**Pull request closed without merging**